axsis.magix.py

```diff
@@ -84,10 +140,23 @@ def on_completed(self):
         pass
 
 
+def create_argparser():
+    parser = argparse.ArgumentParser(description='Start AXSIS Magix client for selected controller.')
+    parser.add_argument('--host', default="localhost", help='Host of controller')
+    parser.add_argument('--port', default=50000, type=int, help='Port of controller')
+    parser.add_argument('--debug', action="store_true", help='Enable debug log')
+    return parser
+
+
+
 def main():
+    args = create_argparser().parse_args()
+    if args.debug:
+        logging.root.setLevel(logging.DEBUG)
+
     loop = asyncio.get_event_loop()
     client = MagixHttpClient(kMagixHost)
-    observer = AxsisObserver(client, loop)
+    observer = AxsisObserver(client, loop, args.host, args.port)
     client.observe(channel=kChannel).pipe(
         ops.filter(lambda event: json.loads(event.data).get('target') == 'axsis'),
```

Comment:
I believe we will need extra filtering here, namely by IP and port

Reply:
What criteria of filtration?

Reply:
By IP and port, these two identify the upstream hardware